HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -47,7 +47,11 @@ def self.calculate(runner, weather, hpxml_bldg, cfa, hvac_systems)
       apply_hvac_heat_pump_logic(hvac_sizing_values, hvac_cooling)
       apply_hvac_equipment_adjustments(runner, hvac_sizing_values, weather, hvac_heating, hvac_cooling, hvac_system)
       apply_hvac_installation_quality(hvac_sizing_values, hvac_heating, hvac_cooling)
+      # apply_hvac_maximum_airflows(hvac_sizing_values, hvac_heating, hvac_cooling)
+      # apply_hvac_airflow_adjustments(hvac_sizing_values, hvac_heating, hvac_cooling)
       apply_hvac_fixed_capacities(hvac_sizing_values, hvac_heating, hvac_cooling)
+      # apply_hvac_blower_fan_adjustment(hvac_sizing_values, hvac_heating, hvac_cooling)
+      apply_hvac_airflow_adjustments(hvac_sizing_values, hvac_heating, hvac_cooling) # <= FIXME should this be before apply_hvac_fixed_capacities?
```

Comment:
In the PR that this supersedes, we had the following methods called in order:
* apply_hvac_maximum_airflows
* apply_hvac_fixed_capacities
* apply_hvac_blower_fan_adjustment

Now that there is a single flag for max airflows and blower fan adjustment, should we have just a single new method for "airflow adjustments" (that's called before or after apply_hvac_fixed_capacities?), or should we follow the same order as before?

Reply:
A single method sounds good to me. I assume it should be after apply_hvac_fixed_capacities.